ClassTranscribeDatabase/Seed.cs

```diff
@@ -305,5 +297,26 @@ public void Seed()
             _context.SaveChanges();
             _logger.LogInformation("Seeded");
         }
+
+        private ApplicationUser newUser(string id, string universityId, string email)
+        {
+            string lower = email.ToLower(), upper = email.ToUpper();
+            ApplicationUser user = new ApplicationUser
+            {
+                Id = "99",
```

Comment:
should this be the given id?
```suggestion
                Id = id,
```